samples/IISSample/project.json

```diff
@@ -9,25 +9,16 @@
   "compilationOptions": {
     "emitEntryPoint": true
   },
-  "commands": {
-    "web": "IISSample"
-  },
   "frameworks": {
     "net451": { },
     "netstandardapp1.5": {
-      "imports": "portable-net451+win8"
+      "dependencies": {
```

Comment:
I imagine there's another wave of updates to change that for all samples but I'm not sure when that's happening.